### PR TITLE
KIALI-1600 [PF4] Create background in grey color as suggested by UX for all views

### PR DESCRIPTION
**Pull request by aljesusg**
UX Design: https://marvelapp.com/i9h9g7j/screen/59315420
Fix : https://github.com/kiali/kiali/issues/1600
Next steps: Continue migration PF4 views

## Overview
overview
#### Istio List
istio_list
#### Service List
services_list
#### Workload List
workloads_list
#### App List
app_list
#### App Details
app_details
#### Service Details
service_details
#### Workload Details
details_w
#### Istio Details
istio_details
##### Logs
logs
#### Metrics
Metrics
#### Traffic
traffic
#### Istio Yaml
Istio_yaml

### CODE REVIEW

**Comment by hhovsepy** [comment]
Actions button in Istio config details is on left side,
The Service details page's Workloads and etc tabs background is partly, how it should be by UX design?

Screenshot from 2019-09-19 10-28-17
Screenshot from 2019-09-19 10-29-16

**Comment by aljesusg** [comment]
> Actions button in Istio config details is on left side,
> The Service details page's Workloads and etc tabs background is partly, how it should be by UX design?
> 
> Screenshot from 2019-09-19 10-28-17
> Screenshot from 2019-09-19 10-29-16

Let me check the Istio view, about details buttons this should be addressed in the Service,Workload PR I think, I can fix it here but my concern is that the PR will be so big ^^

**Comment by aljesusg** [comment]
I'll do the changes here too @hhovsepy

**Comment by lucasponce** [comment]
I think there is an issue with the table for the list.
From the PR:
image
From the Mock:
image

I skip the namespace selector as probably that's a different topic, but if we use the grey background, the table should also adapt to the size of the list, in order to not leave white space if it's empty.

**Comment by lucasponce** [comment]
Same comment as @hhovsepy:

image

Actions would be placed in a different level.
Probably that doesn't need to be included in a single PR as several people are touching that component too.

**Comment by lucasponce** [comment]
Here as well:
image

**Comment by hhovsepy** [comment]
The scrolling is broken, the last line is shown partly:
Screenshot from 2019-09-24 13-07-49

**Comment by aljesusg** [comment]
## NEW VIEWS
## Overview
overview
### Lists
#### Istio List
istio_list
#### Service List
services_list
#### Workload List
workloads_list
#### App List
app_list
### Details
#### App Details
app_details
#### Service Details
service_details
#### Workload Details
details_w
#### Istio Details
istio_details
##### Logs
logs
#### Metrics
Metrics
#1525 
#### Traffic
traffic
#### Istio Yaml
Istio_yaml

**Comment by lucasponce** [comment]
Please @jshaughn @mtho11 take a look to this one.
It's a large PR that touches a lot of files and rebasing can be hard, so take a look as well to see if things looking good.

**Comment by lucasponce** [comment]
This looks non consistent:

image

image

But we have discussed with @jotak that probably we may want to maintain separated, but to me it sounds it should be placed in the same place.

Joel, what do you think ?

**Comment by lucasponce** [comment]
Yes, my point is that now we have an effect with this controllers from past master:

image

image

So, now with this PR, it's more clear that one refresh is out of the tab and other inside.

My thought is how it looks like, if we want to maintain them separate, or if we want to locate them in the same place.

File: src/components/Nav/Page/RenderContent.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { style } from 'typestyle';
+
+const containerPadding = style({ padding: '30px 20px 0 20px' });
+const containerWhite = style({ backgroundColor: '#fff' });
```
[the remark this answers is not in the document]
Comment (aljesusg) [reply src/components/Nav/Page/RenderContent.tsx]: ofc

File: src/components/Nav/Page/RenderHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { style } from 'typestyle';
+
+const containerPadding = style({ padding: '0 20px 20px 20px' });
+const containerWhite = style({ backgroundColor: '#fff' });
```
[the remark this answers is not in the document]
Comment (aljesusg) [reply src/components/Nav/Page/RenderHeader.tsx]: same

File: src/components/Nav/RenderPage.tsx [hunk 3]
```diff
@@ -6,6 +6,8 @@ import { Path } from '../../types/Routes';
 import { style } from 'typestyle';
 
 const containerStyle = style({ marginLeft: 0, marginRight: 0 });
+const containerPadding = style({ padding: '0 20px 0 20px' });
+const containerGray = style({ background: '#f5f5f5' });
```
[the remark this answers is not in the document]
Comment (aljesusg) [reply src/components/Nav/RenderPage.tsx]: same

File: src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx [hunk 4]
```diff
@@ -24,7 +25,13 @@ import { Tab, Text, TextVariants } from '@patternfly/react-core';
 import { dicIstioType } from '../../types/IstioConfigList';
 import { showInMessageCenter } from '../../utils/IstioValidationUtils';
 
-const rightToolbarStyle = style({ float: 'right', marginTop: '8px' });
+const rightToolbarStyle = style({
+  position: 'absolute',
+  right: '20px',
+  zIndex: 1,
+  marginTop: '8px',
+  backgroundColor: 'white'
```
[the remark this answers is not in the document]
Comment (aljesusg) [reply src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx]: same

**Comment by aljesusg** [comment]
> I'm for getting this in ASAP and iterating on any minor issues. But I do see one blocker:
> 
> image

Weird...Is this in FF?

**Comment by jshaughn** [comment]
> Weird...Is this in FF?

No, Chrome 74 on F28.

**Comment by aljesusg** [comment]
> > Weird...Is this in FF?
> 
> No, Chrome 74 on F28.

Let me check it, I use chrome 75 on F30 , I am going to downgrade the browser

**Comment by aljesusg** [comment]
@jshaughn I can't reproduce it, can you make a rebuild and test it again? I did changes in the css and it seems changes are not loaded correctly

**Comment by hhovsepy** [comment]
In distributed tracing tab there is an overlap with namespace selector, otherwise all looks good.
Screenshot from 2019-11-08 11-05-43

**Comment by aljesusg** [comment]
> In distributed tracing tab there is an overlap with namespace selector, otherwise all looks good.
> Screenshot from 2019-11-08 11-05-43

OMG I forgot migrate the tracing tab hahaha let me fix it

**Comment by aljesusg** [comment]
## traces tab
jaeger_traces
# Distributed Tracing Link
jaeger

cc @hhovsepy

**Pull request merged**